Dockerfile

```diff
@@ -0,0 +1,23 @@
+# To build this, from project root run:
+# docker build -t docker.io/djzager/kubevirt-operators -f Dockerfile .
+FROM quay.io/openshift/origin-operator-registry
+
+# Point this at kubevirt/kubevirt when ready
+# This doesn't work because go isn't installed
+# Need root in order to be able to install git
+# Only need this if the manifests are stored elsewhere
+#USER root
+#RUN yum -y install git
+#RUN git clone --branch olm https://github.com/slintes/kubevirt.git && \
```

Comment:
I tried automating the process of grabbing the manifests but since the `operator-registry` image has no Go environment that was a dead end. I could probably do something with multi-stage builds but that was too involved for this initial push.